app/console/commands/Command.js

```diff
@@ -0,0 +1,444 @@
+'use strict';
```

Comment:
As you already know QuorraJS provide support for NodeJS version from 0.12. Hence this should be re written in es5.

This class should go to Positron repo in to the path Positron/lib/console. 
Command service should be attached to app object with the help of a service provider class.
Service provider doc ref: https://quorrajs.org/docs/1.0.0/more/service-providers.html
Service provider example: https://github.com/quorrajs/Positron/blob/1.0/lib/auth/AuthServiceProvider.js

The required services can be loaded to the framework with the help of config defined in app/config/app.js : https://github.com/quorrajs/Quorra/blob/1.0/app/config/app.js#L80